### PR TITLE
bug collapsing overlay on different resolutions

### PR DESCRIPTION
**Pull request by SidelineCory24**
This was not originally collapsing
image

### CODE REVIEW

**Comment by elizabeth-jimenez** [comment]
This works, but im having to click on the very edge of the arrow in order to activate the collapse. It hides behind the pink(added to visualize issue):
image

**Comment by elizabeth-jimenez** [comment]
> This works, but im having to click on the very edge of the arrow in order to activate the collapse. It hides behind the pink(added to visualize issue):
> image

Adding a `margin-top: 35px` to `.bid-tracker-overlay-alert-content` and a `top: 0` to `bid-tracker-alert-container--register` should fix it, but make sure to check all the edges cases (lots with bidtracker)

File: src/sass/_bidTracker.scss [hunk 1]
```diff
@@ -1314,7 +1314,9 @@ $register-hide-icon: 95%;
 }
 
 .bid-tracker-alert-container--register {
-  top: 35px;
+  // top: 35px;
```
Comment (mjoyce91) [inline src/sass/_bidTracker.scss:1317]: comment needed?

File: src/sass/_bidTracker.scss [hunk 2]
```diff
@@ -1314,7 +1314,9 @@ $register-hide-icon: 95%;
 }
 
 .bid-tracker-alert-container--register {
```
Comment (mjoyce91) [inline src/sass/_bidTracker.scss:1316]: Screen Shot 2021-07-19 at 8 48 19 AM

**Review by mjoyce91 — CHANGES_REQUESTED** [review]
Take a look at the mini bid tracker screenshot

File: src/sass/_bidTracker.scss (hunk 1 above)
Comment (SidelineCory24) [reply src/sass/_bidTracker.scss:1317]: removed

File: src/sass/_bidTracker.scss (hunk 2 above)
Comment (SidelineCory24) [reply src/sass/_bidTracker.scss:1316]: update made

**Comment by SidelineCory24** [comment]
disregard last push, still broken

**Review by elizabeth-jimenez — APPROVED** [review]
🍰 🍬 🧁 👍

**Comment by SidelineCory24** [comment]
@mjoyce91 I think you may be the only one who can approve this due to the change request 😅

**Review by mjoyce91 — APPROVED** [review]

**Pull request merged**